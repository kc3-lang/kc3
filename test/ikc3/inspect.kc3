quote inspect({123, %HTTP.Upload{}})
inspect({123, %HTTP.Upload{}})
