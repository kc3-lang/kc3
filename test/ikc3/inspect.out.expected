inspect({123, %HTTP.Upload{}})
"{123, %HTTP.Upload{filename: \"\",\n                   replace: false,\n                   size: ?,\n                   tmp_path: \"\"}}"
